### PR TITLE
Add _UNICODE to default defines on Windows

### LINKED ISSUE #1538: False error
The following code produces an IntelliSense error in VSCode with all the C++ settings at their defaults:

```cpp
#include <windows.h>
#include <tchar.h>

int main()
{
    TCHAR path[MAX_PATH];
    return _tcslen(path);
}
```

Yet, when built with MSVC/Visual Studio,no error is shown by IntelliSense

I'm running Windows 10 build 17093, VSCode 1.19.3, addon version 1.13.1.
Visual Studio is at version 15.5.5

(Looks like it's thinking that I'm trying to use std::experimental::filesystem::v1::path instead of the locally declared `path` array)

### PR DESCRIPTION
**Pull request by sylveon**
Fixes #1538

### CODE REVIEW

**Review by sean-mcmanus — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
You can add an entry to the 0.15.0 CHANGELOG.md if you want.

**Comment by sylveon** [comment]
Thanks, done.

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * Fix error popup appearing with non-workspace files when using `compile_commands.json`. [#1475](https://github.com/Microsoft/vscode-cpptools/issues/1475)
 * Add more macros to `cpp.hint` (fixing missing symbols).
 * Added support for config variables to `c_cpp_properties.json` [#314](https://github.com/Microsoft/vscode-cpptools/issues/314)
+* Define \_UNICODE by default on Windows platforms. [#1538](https://github.com/Microsoft/vscode-cpptools/issues/1538)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: super nit:  Can you use \` around \``_UNICODE`\` instead of '\\'? It will show up in the changelog better that way.

**Review by bobbrow — APPROVED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sylveon) [reply Extension/CHANGELOG.md]: Of course

**Pull request merged**